engine/classes/ElggInstaller.php

```diff
@@ -1573,32 +1573,18 @@ protected function createAdminAccount($submissionVars, $login = false) {
 		$app = $this->getApp();
 
 		try {
-			$guid = register_user(
-				$submissionVars['username'],
-				$submissionVars['password1'],
-				$submissionVars['displayname'],
-				$submissionVars['email']
-			);
+			$user = elgg_register_user([
+				'username' => $submissionVars['username'],
+				'password' => $submissionVars['password1'],
+				'name' => $submissionVars['displayname'],
+				'email' => $submissionVars['email'],
+			]);
```

Comment:
Why not make `$submissionVars` (eg the form) compatible with the required options an then just pass `$submissionVars`

Reply:
not doing this at this moment